### PR TITLE
chore: ☂️ Migrate PreparePrefabHeadersTaskTest.kt to AssertJ

### PR DESCRIPTION
**Pull request by omer-ayhan**
## Summary:
Issue: https://github.com/facebook/react-native/issues/45596

## Changelog:
[INTERNAL] [CHANGED] - Migrate `PreparePrefabHeadersTaskTest.kt` to AssertJ testing library

## Test Plan:
Run `./gradlew -p packages/gradle-plugin test`

### CODE REVIEW

**Comment by omer-ayhan** [comment]
hello @cortinico 👋 this PR is ready for a review🚀

File: packages/gradle-plugin/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/internal/PreparePrefabHeadersTaskTest.kt [hunk 1]
```diff
@@ -41,7 +41,7 @@ class PreparePrefabHeadersTaskTest {
 
     task.taskAction()
 
-    assertTrue(File(outputDir, "sample_library/hello.h").exists())
+    assertThat(File(outputDir, "sample_library/hello.h").exists()).isTrue()
```
Comment (cortinico) [inline packages/gradle-plugin/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/internal/PreparePrefabHeadersTaskTest.kt:44]:
```suggestion
    assertThat(File(outputDir, "sample_library/hello.h")).exists()
```

Comment (cortinico) [reply packages/gradle-plugin/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/internal/PreparePrefabHeadersTaskTest.kt:44]: This applies to all the assertions.

**Review by cortinico — COMMENTED** [review]
Please update the assertions to use `assertThat(File()).exists()`

**Review by cortinico — CHANGES_REQUESTED** [review]
See above comments

**Comment by omer-ayhan** [comment]
changes are done @cortinico

**Review by cortinico — APPROVED** [review]

**Pull request closed without merging**